course6/toyapp_thomasdhuys/drawlib/index.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <iostream>
+#include <cstdint>
+#include <memory>
+
+#include "rgb.h"
+#include "colors.h"
+
+namespace Draw {
+
+struct Index;
+using XCoord = int;
```

Comment:
Surprising type name, especially that you use it also to store Y and Z coordinates.

Reply:
Indeed, bad naming. Renamed to VCoord (Vertex Coordinate)